File: Unidad 2-Estructuras de Control/ejercicio6.java
import java.util.Scanner;
public class Ejercicio6
{
    public static void main(String[] args)
    {
        Scanner reader = new Scanner(System.in);               
        int cateto1, cateto2, hipotenusa;
        System.out.println("Ingrese el valor del primer cateto: ");
        cateto1 = reader.nextInt();

        System.out.println("ingrese el valor del segundo catero: ");
        cateto2 = reader.nextInt();

        System.out.println("Ingrese el valor de la hipotenusa: ");
        hipotenusa = reader.nextInt();
    


    Number hipotenusareal;
        hipotenusareal= Math.sqrt(Math.pow(cateto1, 2) + Math.pow(cateto2, 2));
        
    Number hipotenus, hipotenusarea;
        hipotenusarea=hipotenusareal;
        hipotenus=hipotenusa;
        

    if ( hipotenus == hipotenusarea) {

            System.out.printf ("Los Numeros Si Cumplen Con El Teorema De Pitagoras");
    
    } else {
    
        System.out.printf ("Los Numeros No Cumplen Con El Teorema De Pitagoras");
    
    }
    }
}

File: Unidad 2-Estructuras de Control/Ejercicio6.c
#include <math.h>
#include <stdio.h>

int main() 
{
    double cateto1, cateto2, hipotenusa, hipotenusareal;
     printf("Introduzca el valor del primer cateto: ");
     scanf("%lf", &cateto1);

     printf("Introduzca el valor del segundo cateto: ");
     scanf("%lf", &cateto2);

     printf("Introduzca el valor de la hipotenusa: ");
     scanf("%lf", &hipotenusa);


hipotenusareal= sqrt((cateto1*cateto1) + (cateto2*cateto2));


if (hipotenusa==hipotenusareal) {

       printf("si cumple");
}

else {

    printf("no cumple");
}


}